oesukam/andela-sendit
# [Chore #162532012] Protect create a parcel endpoint

#### What does this PR do?
Protects create a parcel endpoint
#### Description of Task to be completed?
- Check if the user had already confirmed the email before creating a new parcel delivery
- Add email confirmation page

#### How should this be manually tested?
- Register a new user
- Login the new user
- Navigate to create a parcel
- Fill the form then submit
- The parcel submission should fail

#### What are the relevant pivotal tracker stories?
`#162532012`
